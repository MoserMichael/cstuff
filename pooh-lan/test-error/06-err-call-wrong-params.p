

sub foo( a, b )
  return a + b
end


foo( ~c 1 )
