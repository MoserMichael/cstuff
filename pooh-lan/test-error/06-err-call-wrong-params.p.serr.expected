
test-error/06-err-call-wrong-params.p(8): Error: function foo does not have a parameter ~c
      8 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        |foo( ~c 1 )
        |.....^^^^

test-error/06-err-call-wrong-params.p(8): Error: function is called without the required parameter ~a
      8 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        |foo( ~c 1 )
        |.....^^^^

test-error/06-err-call-wrong-params.p(8): Error: function is called without the required parameter ~b
      8 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        |foo( ~c 1 )
        |.....^^^^
Type checking failed
